### PR TITLE
Side channel data and independent node start

### PR DESCRIPTION
**Pull request by cbwl**
This PR introduces two sets of changes:

* First, the ability to decorate `GetOptions` and `RetrievalOptions` with an arbitrary quantity of additional data. There is no specific serialization support provided at this time; the expectation is that data will be serialized and de-serialized by code calling the client API or plugged into the server.
* Second, the ability to invoke `SKAdmin` in such a way that it will generate the start command for a single `DHTNode` and then exit. This is the simplest strategy to support alternative deployment frameworks which already perform the additional functionality that `SKAdmin` provides to orchestrate running processes on remote machines.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,8 @@
-
 ### ide's
 ide
+
 # VS
-.vscode
\ No newline at end of file
+.vscode
+
+# git
```
Comment (bh-ms) [inline .gitignore]: What's the point of this guy?

Comment (cbwl) [reply .gitignore]: These files are created by git during merges -- they are copies of an original file before merge for backup purposes. Documentation is [here](https://git-scm.com/docs/git-mergetool) see the "Temporary Files" section:

> `git mergetool` creates `*.orig` backup files while resolving merges. These are safe to remove once a file has been merged and its `git mergetool` session has completed

Comment (bh-ms) [reply .gitignore]: Ok. Feel free to add to your local .gitignore, but I think we want to leave this out of the main .gitignore.

Comment (cbwl) [reply .gitignore]: I've reverted this.

**Pull request merged**